### ISSUE TITLE
Multilanguage support for frontend

### ISSUE THREAD

**Issue by eproxy** [schachmat/wego#75]
Please add multilanguage support for your frontend. I was able to modify and compile ascii-art-table-.go to translate Morning/Noon/Evening/Night but I don't how to translatate dateFmt function.

**Comment by schachmat** [comment]
There is no possibility from the standard packages, the english weekday and month names are hardcoded. An external library would be needed. Would you mind sharing your changes if they are not just hardcoded replacements to the one language of your choice?

**Comment by eproxy** [comment]
Yes I understand, I was trying to figure it out by myself. To make it consistent you would need to check a value of wwo-lang variable in .wegorc. Create a list of supported langugages, then validate the string against supported languages and if true it would return a reference to a dictionary with a proper translation, 7 days + 12 month is not that much :)  Unless there is a way to interact with OS...

In the attachment times of the day in polish.

[ascii-art-table.go.zip](https://github.com/schachmat/wego/files/149325/ascii-art-table.go.zip)

**Comment by schachmat** [comment]
Hm, I don't think adding it to one frontend is a good idea, since it will not scale. However I can imagine a third kind of plugin just for common translations of the weather conditions, and time strings. The different frontends could use the desired languages translation plugin then to localize the output.

**Comment by dolmen** [comment]
The CLDR has [day of week names](http://cldr.unicode.org/translation/date-time-names).
The [i18n-dateformatter](/altipla-consulting/i18n-dateformatter) project may be helpful to use them.

**Comment by schachmat** [comment]
The project you mentioned does not seem fitting, since it requires the user to generate the code for his locale after installing which further complicates the installation process for new users. Also it does not support translations for the different times of day yet, which we would also want to use for the table headers.
